generator/src/test/java/com/scottlogic/deg/generator/inputs/profileviolation/ProfileViolationTests.java

```diff
@@ -15,7 +15,6 @@
 import com.scottlogic.deg.generator.violations.ViolatedProfile;
```

Comment:
Why are there changes in this file?

Reply:
I was initially getting compile errors due to gradle resolving the jdk differently. I fixed this but the compile issue is because of the javafx Pair class which was removed in later JDKs. Therefore I removed the dependency on the class (see @sladenSL 's comment above).